lib/Conversion/TTNNToEmitC/TTNNToEmitC.cpp

```diff
@@ -616,6 +617,35 @@ class DeallocateOpConversionPattern
   }
 };
 
+// arith::ConstantOp conversion pattern
+//
+class ArithConstantOpConversionPattern
+    : public OpConversionPattern<arith::ConstantOp> {
+
+public:
+  using OpConversionPattern::OpConversionPattern;
+
+  ArithConstantOpConversionPattern(const TypeConverter &typeConverter,
+                                   MLIRContext *context,
+                                   PatternBenefit benefit = 1)
+      : OpConversionPattern<arith::ConstantOp>(typeConverter, context,
+                                               benefit) {}
```

Comment:
An explicit constructor with this behavior is redundant with `using` above it. I see you've mainly used explicit constructors in the `TTNNToEmitC` conversion, I think we are using `using` in most of the other conversions, I would prefer it as it's more concise and immediately suggests that the constructor is almost trivial.